APIBlueprintGenerator.coffee

```diff
@@ -88,6 +88,47 @@ APIBlueprintGenerator = ->
         description: description,
       }
 
+  # Generate a parameter List for the mustache context from a paw Request
+  #
+  # @param [Request] exchange The paw HTTP request
+  #
+  # @return [Object] The template context object
+  #
+  @parameter = (paw_request) ->
+    parameters = []
+    body = paw_request.body
+    has_body = body.length > 0
+    url_parameters = paw_request.getUrlParameters()
+    for key, value of url_parameters
+      parameters.push({
+        name: key,
+        example: value,
+        type: @isNumber(value)
+        })
+    if has_body
```

Comment:
(T_T) Sorry I don't catch what the exact mean....your intention is 

1. I should use the fury.js as SDK for serialize IN ORDER to reduce the duplicate code and make the script more grace and stable.
2. just refix the body parameters as "Attribute" section,then recursing the body to make the attribute tags both in request and response, make sure it is match to API Blueprint syntax.

which one should I do next?

BY THE WAY... I saw you are a member of Apiary,sounds like a company or organization? Did this pull troubled your work ? if not...I am glad to do these works...if it did...I so apologized